### ISSUE TITLE
Markupsafe breaks with latest version of setuptools

### ISSUE THREAD

**Issue by SgtCoDFish** [pallets/markupsafe#116]
The latest version of `setuptools` - https://pypi.org/project/setuptools/46.0.0/ released on 2020-03-08 - removed `Feature` which apparently was a long-deprecated class. There's an issue raised on the repo for setuptools: https://github.com/pypa/setuptools/issues/2017

The following is the output from trying to install markupsafe with the latest version of setuptools.

```text
Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/.../markupsafe/setup.py", line 6, in <module>
        from setuptools import setup, Extension, Feature
    ImportError: cannot import name 'Feature'
```

For now, we can work around this by pinning `setuptools` to the last working version `45.3.0` but I suspect that in the long term markupsafe might need to be updated to remove the use of `Feature`.

This might be something like https://github.com/pallets/markupsafe/issues/23 coming back.

**Comment by davidism** [comment]
This had been fixed since 0.19, but for some reason a revert to that fix made it into 1.0, then was removed again in 1.1.

Unless you *really* need 1.0 for some reason, I'd advise just upgrading to 1.1.1, there shouldn't be any compatibility issues.

**Comment by SgtCoDFish** [comment]
> Unless you really need 1.0 for some reason, I'd advise just upgrading to 1.1.1, there shouldn't be any compatibility issues.

After a bit of an adventure, I got everything upgraded and can confirm that 1.1.1 cures the problem. Thanks very much!

**Comment by advance512** [comment]
Confirm - this happened to me, changing version installed fixed it.

**Comment by jmilagroso** [comment]
> This had been fixed since 0.19, but for some reason a revert to that fix made it into 1.0 ([d2001bb](https://github.com/pallets/markupsafe/commit/d2001bb66b05badc7ac82722e17ddb0e1e81250a)), then was removed again in 1.1.
> 
> Unless you _really_ need 1.0 for some reason, I'd advise just upgrading to 1.1.1, there shouldn't be any compatibility issues.

Thanks @davidism
